test/cri-containerd/container_update_test.go

```diff
@@ -70,14 +72,23 @@ func Test_Container_UpdateResources_CPUShare(t *testing.T) {
 		},
 	}
 
+	// add copies of existing test cases, but enable using annotations to update resources
+	for i, l := 0, len(tests); i < l; i++ {
+		tt := tests[i]
+		tt.requiredFeatures = append(tt.requiredFeatures, featureCRIPlugin)
+		tt.useAnnotation = true
+		tt.name += "_Annotation"
+		tests = append(tests, tt)
+	}
```

Comment:
I think I'd rather you just explicitly duplicate the test cases above with the additional field than to do it like this.